README.rst

```diff
@@ -44,25 +55,76 @@ Install pytorch via your preferred way.
    pip install git+git://github.com/lgvaz/mantisshrimp.git
    pip install -U 'git+https://github.com/cocodataset/cocoapi.git#subdirectory=PythonAPI'
 
-Quick start
------------
+Quick Example: How to train the **Wheat Dataset**
+---------------------------------------------
+
+.. code:: python
+
+   from mantisshrimp.imports import *
+   from mantisshrimp import *
+   import pandas as pd
+   import albumentations as A
+
+   source = Path("/home/lgvaz/.data/wheat")
+   df = pd.read_csv(source / "train.csv")
+   df.head()
+
+   # Custom parser
+   data_splitter = RandomSplitter([.8, .2])
+   parser = WheatParser(df, source / "train")
+   train_rs, valid_rs = parser.parse(data_splitter)
+
+   # shows images with corresponding labels and boxes
+   show_record(train_rs[0], label=False)
+
+   # Transform: supporting albumentations transforms out of the box
+   train_tfm = AlbuTransform([A.Flip()])
+
+   # Create both training and validation datasets
+   train_ds = Dataset(train_rs, train_tfm)
+   valid_ds = Dataset(valid_rs)
+
+   # Create both training and validation dataloaders
+   train_dl = model.dataloader(train_ds, shuffle=True, batch_size=8, num_workers=2)
+   valid_dl = model.dataloader(valid_ds, batch_size=8, num_workers=2)
+
+   # Use pre-trained backbone
+   resnet_101_backbone = MantisFasterRCNN.get_backbone_by_name("resnet101", fpn=True, pretrained=True)
+
+   # Create model
+   model = WheatModel(n_class=2, backbone=resnet_101_backbone)
+
+   # Train (fit) model
+   trainer = Trainer(max_epochs=2, gpus=1)
```

Comment:
I think the last line is missing, `trainer.fit(...)`...

`WheatModel ` is not defined, should we include it's code?

This example is also lightning specific, should we add this information?

I'm working on integration with fastai, should be ready tomorrow, it's something we can add as well =)

Reply:
Ah! I forgot the fit! I was literally half sleeping. Sometimes, I hit the keys while I was snoozing!

Reply:
Great news wrt fastai!